package.json

```diff
@@ -20,7 +20,7 @@
       "url": "https://github.com/toranb/ember-cli-simple-store"
   },
   "engines": {
-    "node": ">= 0.10.0"
+    "node": ">= 0.12.0"
```

Comment:
Lets make this >= 4.0.0 also .. not that we do much in node land explicitly but to avoid any confusion